src/tree/activityLog/ActivityItem.ts

```diff
@@ -30,10 +30,21 @@ export class ActivityItem implements TreeElementBase, Disposable {
 
     public get description(): string | undefined {
         if (this.status === ActivityStatus.Done) {
+            let duration: string | undefined;
+            if (this.activity.startTime && this.activity.endTime) {
+                const startTimeMs: number = this.activity.startTime.getTime();
+                const endTimeMs: number = this.activity.endTime.getTime();
+                duration = dateTimeUtils.getFormattedDurationInMinutesAndSeconds(endTimeMs - startTimeMs);
+            }
+
             if (this.error) {
-                return localize('failed', 'Failed');
+                return duration ?
+                    localize('failedWithDuration', 'Failed ({0})', duration) :
```

Comment:
I think I prefer if it said `Failed in 20s` over the ( ). Nonblocking of course.

Reply:
Do you prefer that for success as well?